tests/test_pytest_plugin.py

```diff
@@ -184,3 +184,14 @@ def test_when_kernel_passed_then_override(pytester: Pytester, testdir2: Never):
     hook_recorder = pytester.inline_run("--nbmake", "--nbmake-kernel=python3")
 
     assert hook_recorder.ret == ExitCode.OK
+
+
+def test_by_cell(pytester: Pytester, testdir2: Never):
```

Comment:
Similar to the comment above, it's ok for a new experimental feature to have thin coverage but I don't want to introduce regressions because I cannot understand the intended feature.